homeassistant/components/smhi/weather.py

```diff
@@ -103,6 +113,13 @@ async def async_setup_entry(
 class SmhiWeather(WeatherEntity):
     """Representation of a weather entity."""
 
+    _attr_attribution = "Swedish weather institute (SMHI)"
+    _attr_temperature_unit = TEMP_CELSIUS
+    _attr_pressure_unit = PRESSURE_HPA
+    _attr_visibility_unit = LENGTH_KILOMETERS
+    _attr_precipitation_unit = LENGTH_MILLIMETERS
+    _attr_wind_speed_unit = SPEED_METERS_PER_SECOND
```

Comment:
This is breaking change.
Unit is possibly now converted for the end-user, to other units.

Reply:
Yes they are. Didn't consider that for the PR comment. I have updated it with a breaking change section and ticket the box for breaking change.

Reply:
However seems as you need to add the breaking change label manually.

Reply:
Honestly, I don't think we should make breaking change for this just yet. There are still architectural discussions going on how to handle the different users for the end-user.

If for some reason that changes, it would become another breaking change down the road.

Reply:
So essentially then leave out the UoM on the three mentioned above so no conversation happens and "fix" the problem once the arch discussion is completed?

Reply:
I think we should avoid breaking changes at this point, yes.

Reply:
Ok, breaking changes has been removed.